aws/resource_aws_backup_vault.go

```diff
@@ -0,0 +1,120 @@
+package aws
+
+import (
+	"regexp"
+
+	"github.com/hashicorp/terraform/helper/validation"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/backup"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsBackupVault() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsBackupVaultCreate,
+		Read:   resourceAwsBackupVaultRead,
+		Update: resourceAwsBackupVaultUpdate,
```

Comment:
Since `resourceAwsBackupVaultUpdate` does not contain any logic, it should be removed and all `Optional`/`Required` attributes marked as `ForceNew: true`.

Reply:
FIxed